pages/linux/protontricks.md

```diff
@@ -18,3 +18,11 @@
 - Run a command within a game's installation directory:
 
 `protontricks -c {{command}} {{appid}}`
+
+- List all installed games:
```

Comment:
```suggestion
- [l]ist all installed games:
```

You can consider option case as more important thing than letter case in code description. That's why you can change `L` to `l` and not use `[L]`.